en-US/Translation.xml

```diff
@@ -92,14 +92,19 @@
         </listitem>
         <listitem>
           <para>
-            Navigation or similar instructions (such as "Navigate to Menu -> Submenu").
+            Navigation or similar instructions (such as "Go to Menu -> Submenu").
```

Comment:
Have other instances of "navigate" already been updated to "go to"?

Reply:
Yes, other occurrences are already changed to "go to".